packages/variable-editor/src/components/variables/master/OverwriteDialog.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  BrowsersView,
+  Button,
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+  useBrowser,
+  type BrowserNode
+} from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { useState, type ReactNode } from 'react';
+import { type Table } from '@tanstack/react-table';
+import type { ProjectVarNode } from '../../../protocol/types';
+import { addNode } from '../../../utils/tree/tree-data';
+import { VariableFactory, type Variable } from '../data/variable';
+import { nodeIcon } from '../data/variable-utils';
+import type { TreePath } from '../../../utils/tree/types';
+import { selectRow } from '../../../utils/table/table';
+import { toRowId } from '../../../utils/tree/tree';
+import { isMetadataType, type MetadataType } from '../data/metadata';
+
+type OverwriteProps = {
+  knownVariables?: ProjectVarNode;
+  table: Table<Variable>;
+  variables: Array<Variable>;
+  setVariables: (variables: Array<Variable>) => void;
+  setSelectedVariablePath: (path: TreePath) => void;
+};
+
+export const OverwriteDialog = ({ knownVariables, table, variables, setVariables, setSelectedVariablePath }: OverwriteProps) => {
+  const toNode = (node: ProjectVarNode): BrowserNode => {
+    const c = node.children.map(child => toNode(child));
+    const icon = nodeIcon(node);
+    const info = node.description;
+    return {
+      value: node.name,
+      info: info,

```

Comment:
detail: no need to do the mapping if constant name is the same as the field name. e.g. `info,` instead of ?`info: info, `, would be enough here